Updated the state name 'OR' in 'IN' Array

sorensd opened the pull request:
Updated the state name of Odisha (previously Orissa) 'IN' array. Cite: https://en.wikipedia.org/wiki/Odisha#Etymology

### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

<!-- Mark completed items with an [x] -->

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Changes proposed in this Pull Request:

<!-- Describe the changes made to this Pull Request and the reason for such changes. -->

Closes # .

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

<!-- Mark completed items with an [x] -->

### Changelog entry

> Tweak - Update state name from `Orissa` to `Odisha`.

sorensd committed:
Updated the state name of 'OR' from India

Updated the state name of Odisha (previously Orissa) 'IN' array. Cite: https://en.wikipedia.org/wiki/Odisha#Etymology

roykho reviewed (commented):
Thanks for the PR however according to several sources, the ISO code did not change. Please see ![](https://d.pr/i/Kr3MnO+) and ![](https://d.pr/i/MsDPQz+)

sorensd committed:
Updated states.php

Updated state name Orissa to Odisha, refer: https://en.wikipedia.org/wiki/Odisha#Etymology

sorensd commented:
Reverted the ISO Code

sorensd commented:
@roykho I reverted the ISO Code to OR, please have a look now. Thanks.

roykho reviewed (approved):
LGTM! Thanks for the PR.
